modules/ROOT/pages/setting-environment-variables.adoc

```diff
@@ -0,0 +1,28 @@
+= Setting Environment Variables
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: anypoint studio, esb, environment variables, system variables, system properties
+
+If you're deploying multiple applications through a Shared Resources structure, then you shouldn't set any properties in the project properties files, as there might potentially be conflicts between the various apps that share a domain. Instead, you can set environment variables over the scope of the deployed app, its domain and other apps under that domain.
```

Comment:
what's a "Shared Resources" structure? Can we link out to info on that or something?

Reply:
"... , as there might potentially be conflicts between..." -> "... because conflicts can occur between..." pls avoid use of "as" in cases like these